guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -2,7 +2,7 @@
 = Installing PostgreSQL
 
 You can install only the same version of PostgreSQL that is installed with the `{foreman-installer}` tool during an internal database installation.
-You can install PostgreSQL using {RHELServer} 7 repositories or from an external source, as long as the version is supported.
+You can install PostgreSQL using {RHEL} 8 repositories or from an external source, as long as the version is supported.
 {Project} supports PostgreSQL version 12.1.
```

Comment:
Unrelated to this change, but we're currently deploying `postgresql-12.11` on EL8, and I guess should just document `12` here?

Reply:
Indeed, PostgreSQL dropped the concept of supported x.y releases with 10. Now they just support x.